### PR TITLE
Mixer Helm improvements

### PR DESCRIPTION
**Pull request by Fructokinase**
1. Remove Recon service group as it is now a Mixer core service.
2. "\." fix
3. Add timestamp to deployment annotation @shifucun  suggested. deploy_gke.sh was run twice in a row to confirm different revisions of the pods were generated.

https://pantheon.corp.google.com/kubernetes/replicaset/us-central1/mixer-us-central1/mixer/mixer-autopush-default-cdbc9d59f/yaml/view?mods=-monitoring_api_staging&project=datcom-mixer-autopush

https://pantheon.corp.google.com/kubernetes/replicaset/us-central1/mixer-us-central1/mixer/mixer-autopush-default-646f65f57/yaml/view?mods=-monitoring_api_staging&project=datcom-mixer-autopush

### CODE REVIEW

File: deploy/README.md [hunk 1]
```diff
@@ -49,8 +49,8 @@ helm upgrade --install mixer-dev deploy/helm_charts/mixer \
     --timeout 10m \
     -f deploy/helm_charts/envs/mixer_dev.yaml \
     --set mixer.githash=$(git rev-parse --short=7 HEAD) \
-    --set-file mixer.schemaConfigs."base.mcf"=deploy/mapping/base.mcf \
-    --set-file mixer.schemaConfigs."encode.mcf"=deploy/mapping/encode.mcf \
+    --set-file mixer.schemaConfigs."base\.mcf"=deploy/mapping/base.mcf \
```
Comment (shifucun) [inline deploy/README.md:52]: what would this cause now?

File: deploy/helm_charts/envs/mixer_autopush.yaml [hunk 2]
```diff
@@ -27,16 +27,6 @@ serviceGroups:
         memoryRequest: "8G"
         memoryLimit: "8G"
       useSearch: true
-  - recon:
```
Comment (shifucun) [inline deploy/helm_charts/envs/mixer_autopush.yaml:30]: Not following why need to remove this? This is to ensure all the recon APIs are routed to the recon pods and not affecting other APIs.

Comment (Fructokinase) [reply deploy/helm_charts/envs/mixer_autopush.yaml]: I was encountering an issue where recon apis were not being detected, looked for the recon flag in main.go and found out that recon is merged into main service. Not remembering that we still wanted separate traffic, I thought recon API group was meant to be removed as well. I reverted this change.

As for the recon api detection issue, will follow up on that separately.

**Review by shifucun — APPROVED** [review]

File: deploy/helm_charts/envs/mixer_autopush.yaml (hunk 2 above)
Comment (Fructokinase) [reply deploy/helm_charts/envs/mixer_autopush.yaml]: ack.

**Pull request merged**